src/components/Currency.tsx

```diff
@@ -7,6 +7,7 @@ import { Body } from './NewTypography';
 
 interface Props {
   amount: string;
+  isNFTAsset?: boolean;
```

Comment:
Use `asset.type`?

Reply:
we're not passing the whole of asset in due to various property edge cases (i.e - currency not being used w/ Asset type sometimes). I removed isNFTAsset from `<Amount />` though, because it's unnecessary there.